cmd/satellite/main.go

```diff
@@ -116,7 +116,7 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		zap.S().Fatal(err)
 	}
 
-	db, err := satellitedb.New(log.Named("db"), runCfg.Database)
+	db, err := satellitedb.New(log.Named("satellite master db"), runCfg.Database)
```

Comment:
Keep this as is, since it comes from satellite log, it should be obvious that it's master db.